docs/input.md

```diff
@@ -1,41 +1,47 @@
 # Input Documentation
 
-## Information about the TSV files
+## General information about the TSV files
 
 Input files for Sarek can be specified using a TSV file given to the `--input` command.
-The TSV file is a Tab Separated Value file with columns:
+There are different kinds of TSV files that can be used as input, depending on the input files available (fastq, uBAM, BAM...).
+For all possible TSV files, described in the next sections, here is an explanation of what the columns refer to:
 
-- `subject sex status sample lane fastq1 fastq2` for step `mapping` with paired-end FASTQs
-- `subject sex status sample lane bam` for step `mapping` with unmapped BAMs
-- `subject sex status sample bam bai recaltable` for step `recalibrate` with BAMs
-- `subject sex status sample bam bai` for step `variantcalling` with BAMs
-
-The content of these columns is quite straight-forward:
-
-- `subject` designate the subject, it should be the ID of the Patient, and it must design only one patient
+- `subject` designates the subject, it should be the ID of the Patient, and it must be unique for each patient
 - `sex` are the sex chromosomes of the Patient, (XX or XY)
-- `status` is the status of the Patient, (0 for Normal or 1 for Tumor)
-- `sample` designate the Sample, it should be the ID of the sample (it is possible to have more than one tumor sample for each patient, i.e. a tumor and a relapse), it must design only one sample
-- `lane` is used when the sample is multiplexed on several lanes, it must be unique for each lane in the same sample
+- `status` is the status of the measured sample, (0 for Normal or 1 for Tumor)
+- `sample` designates the Sample, it should be the ID of the sample (it is possible to have more than one tumor sample for each patient, i.e. a tumor and a relapse), it must be unique for each sample
```

Comment:
Actually if the sample is run on several lanes, then they do need to have the same sample ID to be merged.

Reply:
right, but still the sample ID should be unique for each sample. I will add this info in the `lane` description though.

Reply:
I don't know what to change there actually, it looks pretty clear to me already, but of course open to suggestions :)